test/models/patient_notification_eligibility_test.rb

```diff
@@ -0,0 +1,650 @@
+      ].each do |ineligible_params|
+        patient_args = {
+          preferred_contact_method: preferred_contact_method,
+          pause_notifications: false,
+          purged: false,
+          monitoring: true,
+          closed_at: nil
+        }.merge(ineligible_params)
+        patient = create(:patient, patient_args)
+        assert_ineligible(patient)
+      end
+    end
+  end
+
+  # Below are original tests pulled from the Patient model test
+
+  test 'isolation non reporting send report when latest assessment was more than 1 day ago' do
+    # patient was created more than 24 hours ago
+    patient = create(
+      :patient,
+      preferred_contact_method: 'E-mailed Web Link',
+      monitoring: true,
+      purged: false,
+      isolation: true,
+      created_at: default_days_ago(2)
+    )
+    # patient has asymptomatic assessment more than 24 hours ago but less than 7 days ago
+    create(:assessment, patient: patient, symptomatic: false, created_at: 25.hours.ago)
+    assert_eligible(patient)
+  end
+
+  test 'isolation non reporting send report when no assessments and patient was created more than 1 day ago' do
+    # patient was created more than 24 hours ago
+    patient = create(
+      :patient,
+      preferred_contact_method: 'E-mailed Web Link',
+      monitoring: true,
+      purged: false,
+      isolation: true, created_at: default_days_ago(2)
+    )
+    assert_eligible(patient)
+  end
+
+  test 'exposure send report when latest assessment was more than 1 day ago' do
+    # patient was created more than 24 hours ago
+    patient = create(
+      :patient,
+      preferred_contact_method: 'E-mailed Web Link',
+      monitoring: true,
+      purged: false,
+      isolation: false,
+      created_at: default_days_ago(20),
+      last_date_of_exposure: default_days_ago(14)
+    )
+    # patient has asymptomatic assessment more than 1 day ago but less than 7 days ago
+    create(:assessment, patient: patient, symptomatic: false, created_at: default_days_ago(2))
+    assert_eligible(patient)
+  end
+
+  test 'exposure send report when no assessments and patient was created more than 1 day ago' do
+    # patient was created more than 24 hours ago
+    patient = create(
+      :patient,
+      preferred_contact_method: 'E-mailed Web Link',
+      monitoring: true,
+      purged: false,
+      isolation: false,
+      created_at: default_days_ago(2),
+      last_date_of_exposure: default_days_ago(14)
+    )
+    assert_eligible(patient)
+  end
+
+  test 'exposure send report without continuous exposure' do
+    # patient was created more than 24 hours ago
+    patient = create(
+      :patient,
+      preferred_contact_method: 'E-mailed Web Link',
+      monitoring: true,
+      purged: false,
+      isolation: false,
+      created_at: default_days_ago(4),
+      last_date_of_exposure: default_days_ago(5)
+    )
+    # patient has asymptomatic assessment more than 1 day ago but less than 7 days ago
```

Comment:
Bit confused about these specifics - why does 7 days ago factor in at all here? Same question about symptomatic/asymptomatic?

Reply:
All tests before L312 are eligibility tests that I moved from the patient_test. I'm not sure of the reasoning for the 7 days yet

Reply:
This test was moved from the patient_test. I'm not sure of the reasoning for the 7 days yet

Reply:
@kylietmo here is the git blame.

https://github.com/SaraAlert/SaraAlert/blame/master/test/models/patient_test.rb#L1269